### PR TITLE
Cypress UI tests for the Service Details page

### PR DESCRIPTION
**Pull request by josunect**
Adding Cypress UI tests for the Service Details page

Fixes https://github.com/kiali/kiali/issues/5001

### CODE REVIEW

File: frontend/cypress/integration/common/service_details.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Given('a service in the cluster with a healthy amount of traffic', function () {
+    this.targetNamespace = NAMESPACE;
+    this.targetService = SERVICE;
+});
+
+Then('user sees a list with content {string}', (tab: string) => {
+    cy.get('.pf-c-tabs__list').contains(tab);
+});
+
+Then('user sees productpage details information for service', () => {
```
Comment (hhovsepy) [inline frontend/cypress/integration/common/service_details.ts:18]: I would suggest here to have name as an method argument {string}.

File: frontend/cypress/integration/common/service_details.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+});
+
+function openTab(tab: string) {
+    cy.get('.pf-c-tabs__list').should('be.visible').contains(tab).click();
+}
+
+Then('user sees inbound and outbound traffic information', () => {
+    openTab('Traffic');
+    cy.contains('Inbound Traffic');
+    cy.contains('No Inbound Traffic').should('not.exist');
```
Comment (hhovsepy) [inline frontend/cypress/integration/common/service_details.ts:37]: is it a requirement that traffic should not exist? because the description says "sees"

Comment (josunect) [reply frontend/cypress/integration/common/service_details.ts:37]: But in this case it is looking that the label "No Inbound Traffic" doesn't exist:
image

**Comment by josunect** [comment]
Just a note here: I still would need to add some tests for the "Traces" tab.

File: frontend/cypress/integration/common/service_details.ts [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Then('user sees the actions button', () => {
+    cy.getBySel('wizard-actions').should('be.visible').click();
+    cy.getBySel('wizard-actions').siblings().contains("Request Routing");
+});
+
+Then('user sees {string} details information for service', (name: string) => {
+    cy.get('#ServiceDescriptionCard').within(() => {
+        cy.get('#pfbadge-S').parent().parent().contains(name); // Service
+        cy.get('#pfbadge-A').parent().parent().contains(name); // App
+        cy.get('#pfbadge-W').parent().parent().contains(name + '-v1'); // Workload
```
Comment (hhovsepy) [inline frontend/cypress/integration/common/service_details.ts:32]: one minor thing as well please, add version in params, so this method could be used for all services

**Review by hhovsepy — APPROVED** [review]
lgtm, thank you @josunect

**Pull request merged**